### ISSUE TITLE
Endpoint hit by heroku app does not hash the data

### ISSUE THREAD

**Issue by fishfacemcgee** [Corridors-of-Time-Transcription/puzzlepieces#83]
* Heroku App hits /api/transcriptions ( https://github.com/Corridors-of-Time-Transcription/corridors-of-time/blob/master/pages/api/submit-sequence.js#L5 )
* /api/transcriptions utilizes `TranscriptionDataViewSet` (https://github.com/Corridors-of-Time-Transcription/puzzlepieces/blob/develop/src/collector/urls.py#L8)
* TranscriptionDataViewSet has an active TODO to add hashing https://github.com/Corridors-of-Time-Transcription/puzzlepieces/blob/014c813afb629d23fb14979704883ad719a774f9/src/collector/views.py#L630

NathanADrake in Discord was the initial discoverer of the missing hashes. I poked at a few things and we ended up tracing down the logic flow.

**Comment by stevenzeck** [comment]
Should write a utility to hash the records that aren't hashed too.

**Comment by fishfacemcgee** [comment]
@stevenzeck Agreed. If it helps with whoever ends up writing the tool, the first submission without a hash has a timestamp of `2020-01-19   14:34:12.402780+00:00`.

Edit: to be clear, there **are** submissions past that point with hashes, but that timestamp is where we start getting any hash-less submissions.
